modules/5_install/install.tf

```diff
@@ -84,7 +84,7 @@ locals {
   }
 
   install_inventory = {
-    bastion_hosts  = [for ix in range(length(var.bastion_ip)) : "${var.cluster_id}-bastion-${ix}.${local.cluster_domain}"]
+    bastion_hosts  = [for ix in range(length(var.bastion_ip)) : "${var.name_prefix}-bastion-${ix}"]
```

Comment:
Removing domain will not work for external dns.

Reply:
If the installation failed with domain.  Why just add the domain without cluster_id for external dns? The bastion hostname should be xxxx-bastion-x.`cluster_id`.`cluster_domain`, rIght?